pages/custom-query-modules/manage-query-modules.mdx

```diff
@@ -53,7 +53,7 @@ If you want to reload all existing modules and load any newly added ones, use
 CALL mg.load_all();
 ```
 
-If there is no response (no error message), the load was successful.
+If there is no response (no error message), the load was successful. If you try to reload a module while another thread is using it the reloading will fail with an error message.
```

Comment:
Is it possible for the user to inspect before attempting the reload?

Please give an example of the error message.

Please tell the user how to proceed after seeing the error message.

Reply:
Removed talking about what do the errors mean in this part since it's just an example at the beginning of the page and to avoid duplication the user can check the procedure definition below to see what do the errors mean.